reviewtally/cache/cache_manager.py

```diff
@@ -132,6 +134,93 @@ def cache_per_review(
             f"(TTL: {ttl_desc})",
         )
 
+    def _calculate_pr_list_ttl(self, end_date: datetime) -> int | None:
+        # TTL constants
+        recent_threshold_days = 7
```

Comment:
not happy with the lower case values here and am wondering if a rename to uppercase and hosting up to the top of the file or moving to the init would be better.